frontend/src/lib/instance-views/InstanceView.svelte

```diff
@@ -67,21 +67,23 @@
 		tabindex="0"
 		role="button"
 	>
-		<div class="flex h-10 w-full items-center justify-between pl-4">
-			<div class="text-xs text-grey-light">
-				{entryId}
+		{#if entryId}
```

Comment:
We should change entryId to fall back to undefined probably. Otherwise pretty opaque.

Reply:
not sure what you mean

Reply:
I think you understood you changed to null.